11-prod/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM amazoncorretto:11-alpine
+
+ENV ASUSER='' \
+    UID='' \
```

Comment:
We should add in to the README (based on the standard format for our other image repos), all of these Environment Variables which the user may want to change when using it.

Reply:
Right